### DISCUSSION TITLE
Switch cluster-init to different node in HA cluster?

### DISCUSSION THREAD

**Discussion by ssmall** [k3s-io/k3s#4183] [Q&A]
I have an HA cluster with 3 master nodes. The original node that was started with `cluster-init` has started failing intermittently, and when the other two master nodes can't connect to it there seem to be issues.

I am wondering if it is safe to just swap the systemd command line between the original `cluster-init` node and one of the other two master nodes, or if that will somehow break my cluster. I wasn't able to find anything pertinent to this scenario in the documentation (though maybe I missed something).

**Comment by brandond** [comment]
Once the cluster has been initialized, none of the server nodes actually need the --cluster-init or --server flags, as the etcd server stores the cluster membership internally. Those flags are just necessary for initializing and joining the cluster.
